gnomad_qc/v4/create_release/create_combined_faf_release_ht.py

```diff
@@ -60,8 +61,7 @@ def filter_gene_to_test(ht: hl.Table) -> hl.Table:
 
 
 def extract_freq_info(
-    ht: hl.Table,
-    prefix: str,
+    ht: hl.Table, prefix: str, apply_release_filters: bool = False
```

Comment:
it can be, though in the logs I didn't run it that way since I used --apply-release-filters I think it's best to keep things as true to what was run and only change incorrect logic right now. We can refactor after the tag for the v4.0 release

Reply:
works for me!